src/index.html

```diff
@@ -3,10 +3,43 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Calendar</title>
+    <title>month</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="month month--length-31">
+      <div class="month__day month__day--number-1 month__day--start-day-sun"></div>
+      <div class="month__day month__day--number-2"></div>
+      <div class="month__day month__day--number-3"></div>
```

Comment:
you shouldn't add modificator for each element, only for the parent calendar

Reply:
Not fixed
<img width="745" alt="image" src="https://user-images.githubusercontent.com/99403198/219863880-41b5e026-0984-4bbc-9a13-adb3bea34109.png">
```suggestion
      <div class="month__day month__day--number-1 month__day--start-day-sun"></div>
      <div class="month__day"></div>
      <div class="month__day"></div>
```